### ISSUE TITLE
Any pretrained model available?

### ISSUE THREAD

**Issue by yangchen9** [akirasosa/mobile-semantic-segmentation#17]
could you give a pretrained model for finetune?

**Comment by mayurbhangale** [comment]
Releasing pretrained models-
[](https://drive.google.com/open?id=1DaZPwr4_O5xQ_QRGyzk6_5egk1nZuHhP)

**Comment by ldenoue** [comment]
Thanks for posting. I notice that it only segments the hair.
See my own portrait image which I ran through the predictor:
![laurent_denoue](https://user-images.githubusercontent.com/149561/40908039-f0bae01e-67e5-11e8-854a-d0073f410966.jpg)
![output](https://user-images.githubusercontent.com/149561/40908058-f4ebdbf2-67e5-11e8-9c20-490ab37f92de.jpg)

**Comment by mayurbhangale** [comment]
From readme:

> The goal of this project is to detect hair segments with reasonable accuracy and speed in mobile device. Currently, it achieves 0.89 IoU.

**Comment by ldenoue** [comment]
Thanks, so I'll need to change the training masks to get both classes.
Got it, thanks again.
